packages/file-storage-browser/src/attachment-service.ts

```diff
@@ -0,0 +1,231 @@
+import { bytesBufferToBinaryString, EncryptionKey } from '@meeco/cryppo';
+import * as Common from '@meeco/file-storage-common';
+import { AzureBlockDownload, IFileStorageAuthConfiguration } from '@meeco/file-storage-common';
+import { DirectAttachment } from '@meeco/vault-api-sdk';
+import * as FileUtils from './FileUtils.web';
+
+export class AttachmentService extends Common.AttachmentService {
+  constructor(vaultUrl: string) {
+    super(vaultUrl, fetch);
+  }
+
+  /**
+   * Upload a new attachment. It is encrypted with a random DEK that is returned with the attachment
```

Comment:
Wording here is confusing since it's not a actually encrypted with a random dek - it's encrypted with whatever dek you pass in (you could chose not to use a random dek now)